src/a11y-meta.yml

```diff
@@ -649,6 +733,9 @@
 "responsive-layout":
     title: "Responsive Layout Icon"
     description: "A picture depicting a mobile phone in front of a laptop computer."
+"restricted-area":
+    title: "Restricted Area Icon"
+    description: "A picture depicting a restricted area."
```

Comment:
can we add something like being blocked by a barrier or whatever the thing is called?

Reply:
done